apps/ledger-live-mobile/src/screens/PTX/Earn/index.tsx

```diff
@@ -30,7 +27,7 @@ export const EarnScreen = memo(Earn);
 
 function Earn({ route }: Props) {
   const { theme } = useTheme();
-  const language = useSelector(languageSelector);
+  const { i18 } = useSystem();
```

Comment:
Why don't we get the current language directly from the `useTranslation` hook?

Reply:
I guess because we fetch everything from our Redux Store so I just do the same in this PR. That's a good question tho